lib/raven/configuration.rb

```diff
@@ -210,7 +210,10 @@ def detect_release_from_heroku
     end
 
     def detect_release_from_capistrano
-      File.read(File.join(project_root, 'REVISION')).strip rescue nil
+      version = File.read(File.join(project_root, 'REVISION')).strip rescue nil
+
+      # Capistrano 3+
```

Comment:
If you could please change this comment to reflect your research (3.0 and 3.1 only I guess?)